src/_sass/tilt.scss

```diff
@@ -74,7 +74,7 @@ $tilt2-font-weight-mono-bold: 600;
 
 $tilt2-font-weight-sans-light: 300;
 $tilt2-font-weight-sans-bold: 700;
-$tilt2-font-weight-sans-black: 900;
+$tilt2-font-weight-sans-black: 600;
```

Comment:
This sans-black variable is used in the mixin `font-hero`, which is used for the big bold text on the homepage and enterprise page. So changing the value here has unintended effects.

A better way to do "H: Change weight of H1" might be to use the lighter weight font in docs.scss:

```
.Docs-content-title {
  h1 {
    font-weight: $tilt2-font-weight-sans-bold;
```

The assumption I made here is that we're restricting ourselves to just a few weights of the sans font, which is why I made just 3 weight variables. But, if you think it's a pattern that we want to use other weights, too, we might want to add another weight.